docs/source/tutorials/configuration.rst

```diff
@@ -272,7 +272,7 @@ executing
    manim -o myscene -c RED <file.py> SceneName
 
 There is also a **library-wide** config file that determines manim's default
-behavior, and applies to every user of the library.  It has the least
+behavior and applies to every user of the library.  It has the least
```

Comment:
a case for keeping the comma as it is *could* be made here.

Reply:
Well it would be grammatically wrong then

Reply:
> Well it would be grammatically wrong then

you cannot assert such a thing. This is a question of style here.

Reply:
Ok so should add or remove it then?

Reply:
nah. your suggestion is ok. I was just complaining about the tone of your statement ;)

Reply:
Oh ok sorry for that